ecosystem/typescript/sdk_v2/tests/e2e/api/general.test.ts

```diff
@@ -0,0 +1,81 @@
+import { AptosConfig, Aptos } from "../../../src";
+import { ViewRequest } from "../../../src/types";
+import { Network } from "../../../src/utils/api-endpoints";
+
+describe("general api", () => {
+  test("it fetches ledger info", async () => {
+    const config = new AptosConfig({ network: Network.LOCAL });
+    const aptos = new Aptos(config);
+    const ledgerInfo = await aptos.getLedgerInfo();
+    expect(ledgerInfo.chain_id).toBe(4);
+  });
+
+  test("it fetches chain id", async () => {
+    const config = new AptosConfig({ network: Network.LOCAL });
+    const aptos = new Aptos(config);
+    const chainId = await aptos.getChainId();
+    expect(chainId).toBe(4);
+  });
+
+  test("it fetches block data by block height", async () => {
+    const config = new AptosConfig({ network: Network.LOCAL });
+    const aptos = new Aptos(config);
+    const blockHeight = 100;
+    const blockData = await aptos.getBlockByHeight({ blockHeight });
+    expect(blockData.block_height).toBe(blockHeight.toString());
```

Comment:
Interesting, you should probably make this block a guaranteed block, like block 0 or 1. rather than 100.

1 is small, so I'd probably do that.